planterbox_webdriver/screenshot.py

```diff
@@ -45,13 +45,26 @@ def capture_screenshot_delay(test, delay):
 
 @hook('before', 'feature')
 def set_save_directory(test):
-    """Sets the root save directory for saving screenshots.
+    """
+        Sets the root save directory for saving screenshots.
+    """
+    
+    import datetime
+    import pytz
+    
+    now = datetime.datetime.now(pytz.utc)
+    test_run_date = now.strftime("%Y-%m-%d")
```

Comment:
This would mean all the screenshots from every test on the same day get dumped into a single directory. That seems less than ideal?